packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -215,10 +215,15 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
     return currentPosition;
   }
   private _updateValue(value: number, renderedValue: number): void {
-    const interval: number = 1.0 / this.props.step!;
-    // Make sure value has correct number of decimal places based on steps without JS's floating point issues
-    const roundedValue: number = Math.round(value * interval) / interval;
+    const { step } = this.props;
+
+    let numDec = 0;
+    if (Math.floor(step!) !== step!) {
```

Comment:
`!` is a bit of a red flag for me. What happens here if step is undefined / null?